bumble/core.py

```diff
@@ -28,11 +28,19 @@
 # -----------------------------------------------------------------------------
 # fmt: off
 
-BT_CENTRAL_ROLE    = 0
-BT_PERIPHERAL_ROLE = 1
+class PhysicalTransport(enum.IntEnum):
+    BR_EDR = 0
+    LE     = 1
 
-BT_BR_EDR_TRANSPORT = 0
-BT_LE_TRANSPORT     = 1
+class Role(enum.IntEnum):
+    CENTRAL    = 0
+    PERIPHERAL = 1
+
+BT_CENTRAL_ROLE    = Role.CENTRAL
+BT_PERIPHERAL_ROLE = Role.PERIPHERAL
+
+BT_BR_EDR_TRANSPORT = PhysicalTransport.BR_EDR
+BT_LE_TRANSPORT     = PhysicalTransport.LE
 
```

Comment:
what is the reason for these defines? could they be removed?

Reply:
That's for backward compatibility. There have been an significant usage of these primitive constants, so removing them here will be irresponsible.